website/docs/r/ldap_verify.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: ldap-verify"
+sidebar_current: "docs-mongodbatlas-resource-ldap-verify"
+description: |-
+    Provides a LDAP Verify resource.
+---
+
+# mongodbatlas_ldap_verify
+
+`mongodbatlas_ldap_verify` provides an LDAP Verify resource. This allows ldap verify to be created.
```

Comment:
Let's edit this one too:
`mongodbatlas_ldap_verify` provides an LDAP Verify resource.  This allows a  a verification of an LDAP configuration over TLS for an Atlas project. Atlas retains only the most recent request for each project.